can/interfaces/ixxat/canlib.py

```diff
@@ -1018,6 +990,7 @@ def bus_load(self) -> int:
             warnings.warn("The current adapter does not support bus load measurement")
             return 0
 
+    @property
```

Comment:
[test_interface_ixxat.py#L179](https://github.com/zariiii9003/python-can/blob/7ab5f586b7dd087a0bd12d47a39438580ce4a3d1/test/test_interface_ixxat.py#L179) also needs to be changed to remove the brackets from the function call to remain compatible with this becoming a property.

Reply:
Done